Gauge.VisualStudio.TestAdapter/GaugeRunner.cs

```diff
@@ -189,6 +200,18 @@ private void OnOutputDataReceived(object sender, DataReceivedEventArgs args)
             }
         }
 
+        private static string EscapeCharacterSequence(string data)
+        {
+            data = Regex.Replace(data, @"([^\\])\\([^\\])", "$1\\\\$2");
```

Comment:
you don't need regex for string replace, `String.Replace` would be better, and performant.

If you need to use regex, you can have one regex for all characters to be escaped, and not one per character.

Also, if you are using regex, please use `RegexOptions.Compiled` (ref: https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regexoptions?view=netframework-4.8#fields)

Reply:
This whole scaping bit has been removed from `gauge-visualstudio`. Now **gauge** will take care of proper JSON encoding.